pillar/fluentd/init.sls

```diff
@@ -1,3 +1,10 @@
+{% set ENVIRONMENT = salt.grains.get('environment') %}
+{% set minion_id = salt.grains.get('id', '') %}
+{% set cert = salt.vault.cached_write('pki-intermediate-{}/issue/fluentd-client'.format(ENVIRONMENT), common_name='fluentd.{}.{}'.format(minion_id, ENVIRONMENT)) %}
+{% set fluentd_cert_path = salt.sdb.yaml.get('fluentd:cert_path') %}
```

Comment:
We don't have to use `sdb.get` in jinja as Salt supports passing an SDB string directly in most of its files. So this would end up being:

```sls
cert:
  fluentd_cert:
    content: |
      {{ cert.data.certificate | indent(8) }}
    path: sdb://yaml/fluentd:cert_path
```

Reply:
Fixed call to sdb, but left the `cert` config as is, since it feels cleaner to make changes to a set variable as opposed to dig through the whole file (granted this is a small file) and make changes somewhere in there.